pennylane/ops/qutrit/channel.py

```diff
@@ -313,3 +313,119 @@ def compute_kraus_matrices(gamma_1, gamma_2):  # pylint:disable=arguments-differ
             math.cast_like(math.array([[0, 0, 1], [0, 0, 0], [0, 0, 0]]), gamma_2), gamma_2
         )
         return [K0, K1, K2]
+
+
+class TritFlip(Channel):
```

Comment:
I am not sure about the name and would like to know what you all think.

Reply:
I like it!